### ISSUE TITLE
fine-tune parameters whisper for reproduction

### ISSUE THREAD

**Issue by abcnorio** [AI4Bharat/Svarah#5]
Hello,

can you please add and publish your fine-tune parameters for whisper? I tried to reproduce your results and this was not possible, esp. `wer` using a fine-tune script from [vasistalodagala](https://github.com/vasistalodagala/whisper-finetune). Unfortunately your paper does not cite those values nor your webpage.

Esp.

- learning rate
- number of steps (or epochs)
- any regularization parameters against overfitting, ie. drop-out, weight decay, etc.
- grad_acc and batch size (train, eval)
- ratio train vs. eval sample size

Thanks.
